src/Plugin/LessPlugin.php

```diff
@@ -25,6 +27,15 @@ public function activate(PluginApi $plugin_api): void
         if ($plugin_api->getConfig()->isDev()) {
             $less_collector = new CachedImportCollector($less_collector, $plugin_api->getCache());
         }
+        $plugin_api->getConfig()->getEventDispatcher()->addListener(
+            AssetEvents::FIND_SOURCE,
+            function (FindSourceEvent $ev) {
+                $file_path = preg_replace('/\.css$/', '.less', $ev->getFilepath());
```

Comment:
Why wouldn't the Angular component refer to the `.less` file immediately? That would also help the IDE to find the file.

Reply:
if Angular Ahead of Time would be used, using .less inmediately would load the less file in as css.